pylintrc

```diff
@@ -70,7 +70,8 @@ disable=
     standarderror-builtin,import-star-module-level,old-octal-literal,
     metaclass-assignment,reduce-builtin,indexing-exception,
     parameter-unpacking,long-suffix,cmp-method,using-cmp-argument,
-    useless-suppression
+    useless-suppression,import-error,
+    empty-docstring,missing-docstring,redefined-outer-name
```

Comment:
pylint is stupid and does not understand that `__init__` calls methods that in turn set attributes (i.e. when widgets use methods to construct gui)